src/chef_ci_status.rb

```diff
@@ -6,13 +6,44 @@
 require 'octokit'
 require 'set'
 
-# Get GitHub token
+# Load configuration
+begin
+  require_relative '../config/initializers/config'
+rescue LoadError => e
+  puts "ERROR: Configuration system not loaded: #{e.message}"
+  puts 'The config gem may not be installed.'
+  puts "Run 'bundle install' to install dependencies."
+  puts 'Using hardcoded defaults.'
+rescue StandardError => e
+  puts "ERROR: Configuration failed to load: #{e.message}"
+  puts 'Check your configuration files for syntax errors or'
+  puts 'missing required fields.'
+  puts 'Using hardcoded defaults.'
+end
```

Comment:
I don't think this is necessary. I wrote this... yesterday. We don't have to worry about strong back-compat to that one user Phil. :)